### ISSUE TITLE
Travis tests are disabled?

### ISSUE THREAD

**Issue by namedgraph** [perma-id/w3id.org#1629]
It looks like `travis.yml` is all commented out and not testing anything?
https://travis-ci.org/perma-id/w3id.org/jobs/648154777/config?utm_medium=notification&utm_source=github_status

Our PR passed but we got non-working links anyway.

**Comment by dgarijo** [comment]
Can you please specify which links are not working? Travis has not been working for a while, and should not be a reference. For example https://w3id.org/atomgraph/linkeddatahub/domain seems to work fine

**Comment by davidlehn** [comment]
Yes, I disabled that long ago because it was causing too much trouble.  For every PR it was scanning for all w3id.org URLs in all READMEs.  Peoples redirects rot over time so there are always a bunch of failing URLs.  That caused everyone to get confused.  Also the link checker itself had some odd TLS network problems on the travis infrastructure, and I failed to debug it.  If someone would like to update the script to just check the files in a PR, and figure out why the link checker is sometimes broken, that would be great, and we'll re-enable PR testing.
